From 0f84666385b03b6a7b4e99928339e33fae00da83 Mon Sep 17 00:00:00 2001
From: Jenkins <dev-kitchen@cloudera.com>
Date: Thu, 19 Jan 2017 10:52:58 -0800
Subject: [PATCH 219/219] Branching for 5.8.4 on Thu Jan 19 10:52:41 PST 2017

JOB_NAME : 'Cut-Release-Branches'
BUILD_NUMBER : '450'
CODE_BRANCH : ''
OLD_CDH_BRANCH : 'cdh5_5.8.x'

Pushed to remote apache	git@github.sf.cloudera.com:Mirrors/apache-hbase.git (push)
origin	git@github.sf.cloudera.com:CDH/hbase.git (push)
---
 hbase-annotations/pom.xml                |    2 +-
 hbase-assembly/pom.xml                   |    2 +-
 hbase-checkstyle/pom.xml                 |    4 ++--
 hbase-client/pom.xml                     |    2 +-
 hbase-common/pom.xml                     |    2 +-
 hbase-examples/pom.xml                   |    2 +-
 hbase-external-blockcache/pom.xml        |    2 +-
 hbase-hadoop-compat/pom.xml              |    2 +-
 hbase-hadoop2-compat/pom.xml             |    2 +-
 hbase-it/pom.xml                         |    2 +-
 hbase-prefix-tree/pom.xml                |    2 +-
 hbase-procedure/pom.xml                  |    2 +-
 hbase-protocol/pom.xml                   |    2 +-
 hbase-resource-bundle/pom.xml            |    2 +-
 hbase-rest/pom.xml                       |    2 +-
 hbase-server/pom.xml                     |    2 +-
 hbase-shaded/hbase-shaded-client/pom.xml |    2 +-
 hbase-shaded/hbase-shaded-server/pom.xml |    2 +-
 hbase-shaded/pom.xml                     |    2 +-
 hbase-shell/pom.xml                      |    2 +-
 hbase-spark/pom.xml                      |    2 +-
 hbase-testing-util/pom.xml               |    2 +-
 hbase-thrift/pom.xml                     |    2 +-
 pom.xml                                  |    4 ++--
 24 files changed, 26 insertions(+), 26 deletions(-)

diff --git a/hbase-annotations/pom.xml b/hbase-annotations/pom.xml
index 6bad328..c464098 100644
--- a/hbase-annotations/pom.xml
+++ b/hbase-annotations/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-assembly/pom.xml b/hbase-assembly/pom.xml
index 0802300..6f46eba 100644
--- a/hbase-assembly/pom.xml
+++ b/hbase-assembly/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-assembly</artifactId>
diff --git a/hbase-checkstyle/pom.xml b/hbase-checkstyle/pom.xml
index e70390b..74f5912 100644
--- a/hbase-checkstyle/pom.xml
+++ b/hbase-checkstyle/pom.xml
@@ -24,14 +24,14 @@
 <modelVersion>4.0.0</modelVersion>
 <groupId>org.apache.hbase</groupId>
 <artifactId>hbase-checkstyle</artifactId>
-<version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+<version>1.2.0-cdh5.8.4</version>
 <name>Apache HBase - Checkstyle</name>
 <description>Module to hold Checkstyle properties for HBase.</description>
 
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-client/pom.xml b/hbase-client/pom.xml
index 8c7a5e5..8b4ba59 100644
--- a/hbase-client/pom.xml
+++ b/hbase-client/pom.xml
@@ -24,7 +24,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-common/pom.xml b/hbase-common/pom.xml
index 3f429d6..25f3983 100644
--- a/hbase-common/pom.xml
+++ b/hbase-common/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-examples/pom.xml b/hbase-examples/pom.xml
index d1f0753..89e49a0 100644
--- a/hbase-examples/pom.xml
+++ b/hbase-examples/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-examples</artifactId>
diff --git a/hbase-external-blockcache/pom.xml b/hbase-external-blockcache/pom.xml
index 4c169d5..7a48ea3 100644
--- a/hbase-external-blockcache/pom.xml
+++ b/hbase-external-blockcache/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-external-blockcache</artifactId>
diff --git a/hbase-hadoop-compat/pom.xml b/hbase-hadoop-compat/pom.xml
index 9c199e5..5f1fd8c 100644
--- a/hbase-hadoop-compat/pom.xml
+++ b/hbase-hadoop-compat/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-hadoop2-compat/pom.xml b/hbase-hadoop2-compat/pom.xml
index d1dccb2..fcefd78 100644
--- a/hbase-hadoop2-compat/pom.xml
+++ b/hbase-hadoop2-compat/pom.xml
@@ -21,7 +21,7 @@ limitations under the License.
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-it/pom.xml b/hbase-it/pom.xml
index 7d093af..b38013a 100644
--- a/hbase-it/pom.xml
+++ b/hbase-it/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-prefix-tree/pom.xml b/hbase-prefix-tree/pom.xml
index 5d91557..dfcde83 100644
--- a/hbase-prefix-tree/pom.xml
+++ b/hbase-prefix-tree/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-procedure/pom.xml b/hbase-procedure/pom.xml
index 2cab6f3..9aa8a81 100644
--- a/hbase-procedure/pom.xml
+++ b/hbase-procedure/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-protocol/pom.xml b/hbase-protocol/pom.xml
index 1057d44..dfabe1c 100644
--- a/hbase-protocol/pom.xml
+++ b/hbase-protocol/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
 
diff --git a/hbase-resource-bundle/pom.xml b/hbase-resource-bundle/pom.xml
index c26a0e0..6b14ae5 100644
--- a/hbase-resource-bundle/pom.xml
+++ b/hbase-resource-bundle/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
 
diff --git a/hbase-rest/pom.xml b/hbase-rest/pom.xml
index 17f6271..bf07003 100644
--- a/hbase-rest/pom.xml
+++ b/hbase-rest/pom.xml
@@ -25,7 +25,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-rest</artifactId>
diff --git a/hbase-server/pom.xml b/hbase-server/pom.xml
index 6518492..9535068 100644
--- a/hbase-server/pom.xml
+++ b/hbase-server/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-server</artifactId>
diff --git a/hbase-shaded/hbase-shaded-client/pom.xml b/hbase-shaded/hbase-shaded-client/pom.xml
index 02b116f..0128085 100644
--- a/hbase-shaded/hbase-shaded-client/pom.xml
+++ b/hbase-shaded/hbase-shaded-client/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-client</artifactId>
diff --git a/hbase-shaded/hbase-shaded-server/pom.xml b/hbase-shaded/hbase-shaded-server/pom.xml
index 2e88ead..d8cb82b 100644
--- a/hbase-shaded/hbase-shaded-server/pom.xml
+++ b/hbase-shaded/hbase-shaded-server/pom.xml
@@ -24,7 +24,7 @@
     <parent>
         <artifactId>hbase-shaded</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded-server</artifactId>
diff --git a/hbase-shaded/pom.xml b/hbase-shaded/pom.xml
index 008bcca..3c48d93 100644
--- a/hbase-shaded/pom.xml
+++ b/hbase-shaded/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-shaded</artifactId>
diff --git a/hbase-shell/pom.xml b/hbase-shell/pom.xml
index 0d6d72b..166a0c1 100644
--- a/hbase-shell/pom.xml
+++ b/hbase-shell/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-shell</artifactId>
diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index b4e7ab7..673ba54 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -30,7 +30,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-spark</artifactId>
diff --git a/hbase-testing-util/pom.xml b/hbase-testing-util/pom.xml
index f23b468..ac5c0f1 100644
--- a/hbase-testing-util/pom.xml
+++ b/hbase-testing-util/pom.xml
@@ -23,7 +23,7 @@
     <parent>
         <artifactId>hbase</artifactId>
         <groupId>org.apache.hbase</groupId>
-        <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+        <version>1.2.0-cdh5.8.4</version>
         <relativePath>..</relativePath>
     </parent>
     <artifactId>hbase-testing-util</artifactId>
diff --git a/hbase-thrift/pom.xml b/hbase-thrift/pom.xml
index b0d51b1..de7ad46 100644
--- a/hbase-thrift/pom.xml
+++ b/hbase-thrift/pom.xml
@@ -23,7 +23,7 @@
   <parent>
     <artifactId>hbase</artifactId>
     <groupId>org.apache.hbase</groupId>
-    <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+    <version>1.2.0-cdh5.8.4</version>
     <relativePath>..</relativePath>
   </parent>
   <artifactId>hbase-thrift</artifactId>
diff --git a/pom.xml b/pom.xml
index 0bbfed5..a895de3 100644
--- a/pom.xml
+++ b/pom.xml
@@ -32,14 +32,14 @@
   <parent>
     <groupId>com.cloudera.cdh</groupId>
     <artifactId>cdh-root</artifactId>
-    <version>5.8.4-SNAPSHOT</version>
+    <version>5.8.4</version>
     <relativePath/>
     <!-- no parent resolution -->
   </parent>
   <groupId>org.apache.hbase</groupId>
   <artifactId>hbase</artifactId>
   <packaging>pom</packaging>
-  <version>1.2.0-cdh5.8.4-SNAPSHOT</version>
+  <version>1.2.0-cdh5.8.4</version>
   <name>Apache HBase</name>
   <description>
     Apache HBase is the Hadoop database. Use it when you need
-- 
1.7.9.5

